device_samples/getting_started/README.md

```diff
@@ -0,0 +1,38 @@
+# Getting started samples (level 100)
+
+These getting started samples are perfect for someone just getting into the [Azure IoT Device SDK for Node.js](https://github.com/Azure/azure-iot-sdk-node). The samples are targeted, simple, and you should be able to get them up and running in just a couple minutes. For more complicated samples and scenarios, see the [How to guides](../how_to_guides) section.
```

Comment:
I think we should emphasize that these level 100 samples are not intended to be the basis of a production application